.github/workflows/acceptance_tests_base.yml

```
@@ -213,6 +213,19 @@ jobs:
       - name: Run acceptance tests
         if: ${{ !inputs.skip_tests && needs.filters.outputs.require_acceptance_tests == 'true' }}
         run: ./testsuite/podman_runner/${{ inputs.tests }}
+      
+      - name: Upload Cucumber JSON Reports
```

Actually this step only make sense if the acceptance tests runs, so I would include the same condition than above:

```suggestion
      - name: Upload Cucumber JSON Reports
      if: ${{ !inputs.skip_tests && needs.filters.outputs.require_acceptance_tests == 'true' }}
```